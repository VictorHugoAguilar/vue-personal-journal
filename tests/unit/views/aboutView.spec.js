import {shallowMount} from "@vue/test-utils";
import AboutView from '@/views/AboutView';

describe("about view component", () => {
    test("debe renderizar el componente correctamente", () => {
        const wrapper = shallowMount(AboutView);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
