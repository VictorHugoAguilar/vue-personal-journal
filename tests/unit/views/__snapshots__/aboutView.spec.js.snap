// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about view component debe renderizar el componente correctamente 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
